jax/_src/tree_util.py

```diff
@@ -561,13 +558,6 @@ def pprint(self) -> str:
   def __str__(self):
     return self.pprint()
 
-class FlattenedKeyPathEntry(_DeprecatedKeyPathEntry):  # fallback
-  def pprint(self) -> str:
-    return f'[<flat index {self.key}>]'
-  def __str__(self):
-    return self.pprint()
-
-
```

Comment:
This is a non-public class, and is currently not used anywhere, so it's safe to delete.